Tests/Singulink.Globalization.Currency.Tests/ImmutableSortedMoneySetTests/RemoveTests.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System.Collections.Generic;
+using System.Collections.Immutable;
+using Shouldly;
+
+namespace Singulink.Globalization.Tests.ImmutableSortedMoneySetTests;
+
+[TestClass]
+public class RemoveTests
+{
+    private static readonly Money Usd100 = new(100m, "USD");
+    private static readonly Money Cad50 = new(50m, "CAD");
+    private static readonly Money Eur25 = new(25m, "EUR");
+    private static readonly ImmutableSortedMoneySet _immutableSet = [Usd100, Cad50, Eur25];
+
+    // public bool Remove(string currencyCode) tests
+
+    [TestMethod]
+    public void RemoveCurrencyCode_CurrencyFound_RemovesValueAndReturnsTrue()
```

Comment:
These operations don't return true/false, they return an updated set, so I would just remove the "ReturnsTrue" and just leave it at "RemovesValue". Adjust remaining method names accordingly.